tests/app/questionnaire/test_router.py

```diff
@@ -206,17 +227,50 @@ def test_return_to_final_summary_next_location_url(self):
             schema, self.answer_store, self.list_store, progress_store, self.metadata
         )
         current_location = Location(section_id="default-section", block_id="name-block")
+        routing_path = RoutingPath(["name-block"], section_id="default-section")
+        next_location = router.get_next_location_url(
+            current_location, routing_path, return_to="final-summary"
+        )
+
+        self.assertEqual(url_for("questionnaire.submit_questionnaire"), next_location)
```

Comment:
I think a different schema with more than one block should be used for this test. Currently the `next_location` is `questionnaire.submit_questionnaire` with or without the `return_to` being set.

Reply:
Now uses a schema with multiple blocks.